src/map/battle.cpp

```diff
@@ -411,7 +416,12 @@ int battle_delay_damage(t_tick tick, int amotion, struct block_list *src, struct
 	dat->additional_effects = additional_effects;
 	dat->src_type = src->type;
 	dat->isspdamage = isspdamage;
-	if (src->type != BL_PC && amotion > 1000)
+	if (skill_id == 0 && src->type == BL_MOB && battle_config.synchronize_damage && amotion > status_get_clientamotion(src)) {
```

Comment:
```suggestion
	if (battle_config.synchronize_damage && skill_id == 0 && src->type == BL_MOB && amotion > status_get_clientamotion(src)) {
```
Feature check first, so that it fails as fast as possible.